### PR TITLE
View property

### PR DESCRIPTION
**Pull request by pngtiff**

### CODE REVIEW

File: view/viewProfile.php [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 <?php ob_start();?>
-<section>  <!-- RENAMED "main" into "section" for consistency and merged the css -->
-    <div>
-        <button id='homeButton'><a href='index.php'>Home</a></button>
-    </div>
-    <div id="profileContainer" class="flexColumn">
-        <div id='userInfo' class="flexColumn">
-            <div id='userMain'>
-                <div><div id='userImgContainer'><img id='userImg' src="<?= $user['profile_img'];?>" alt="<?= $user['id'].'profile image';?>"></div></div>
+<main>
```
Comment (matthieuauvy) [inline view/viewProfile.php]: Change to section

File: public/style/style.css [hunk 2]
```diff
@@ -137,30 +196,58 @@ section
...
+
+.errorText {
+    visibility: hidden;
+}
 
 
 
 /* PROFILE VIEW */
 
+main {
```
Comment (matthieuauvy) [inline public/style/style.css]: delete and merge with section

**Review by matthieuauvy — CHANGES_REQUESTED** [review]

File: view/viewProfile.php (hunk 1 above)
Comment (pngtiff) [reply view/viewProfile.php]: done

File: public/style/style.css (hunk 2 above)
Comment (pngtiff) [reply public/style/style.css]: done

**Review by matthieuauvy — APPROVED** [review]

**Pull request merged**